### DISCUSSION TITLE
links aren't working on this site only for me

### DISCUSSION THREAD

**Discussion by gsgoldma** [civitai/civitai#256] [Ideas]
broswer is firefox:
Maybe this is a firefox problem, but if I click a link it does nothing except update my address bar. I have to manually hit enter on my address bar or click open link in new tab for it to execute the link.
It works on microsoft edge fine.

I've cleared my cache already, and this doesn't happen to any other site with me.

**Comment by JustMaier** [comment]
Hey we had some server issues around the time that you posted this. They were resolved 2 or so hours later. Are you still having trouble?

**Reply by gsgoldma** [reply]
yes still. been happening for about a week now

**Reply by manuelurenah** [reply]
Hey @gsgoldma, have you tried other browsers? Also please check while in incognito mode, it may be that an extension is blocking the links or navigation in the app.

**Reply by gsgoldma** [reply]
it does work in browsers other than firefox
not in incognito mode
persisted after disabling extensions too

**Reply by manuelurenah** [reply]
That's so odd, I have tested with firefox and seems to be working fine. Did you try updating to the latest version?

**Comment by WASasquatch** [comment]
This does happen often. The sites content loading sometimes bugs and nothing works. Like clicking the CivitAI logo to go to the homepage, it will just update the address bar, but the page doesn't change, but clicking refresh, or hitting enter on address bar will go to the page the the address bar updated to.
